scripts/install

```diff
@@ -71,14 +71,22 @@ exec_as_app console.php database:configure "$app" "$db_pwd" "$app"
 # Create database contents
 exec_as_app console.php database:install
 
+# Set default database directory
+FGALDIR=/home/yunohost.app/$app
```

Comment:
there is already a variable name for the `/home/yunohost.app/$app` path:
`$data_dir="/home/yunohost.app/$app"`